templates/song_rater/song_list.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Song_List</title>
+</head>
+<body>
+<h1>List of all Songs</h1>
+<table border="1" width="600">
+<tr>
+<td>Title</td>
+<td>Author</td>
+</tr>
+
+{% for song in object_list%}
+<tr>
+<td>{{ song.title }}</td>
+<td>{{ song.author }}</td>
+</tr>
+{% endfor %}
```

Comment:
Wouldn't it be nice to display something here in case there are no songs in the list yet?